sig-scalability/thresholds.md

```diff
@@ -42,7 +42,7 @@ Important notes about the numbers:
 | Total number of all objects         | 250000         |             | 1000000        |
 | Number of nodes                     | 5000           |             | 5000           |
 | Number of pods                      | 150000         |             | 500000         |
-| Number of pods per node<sup>1</sup> | 100            |             | 100            |
+| Number of pods per node<sup>1</sup> | 110            |             | 500            |
```

Comment:
I agree with 110.

I have heard any commitment that we will ever want to get to 500 and make it official.

Reply:
Yes, RedHat is dealing with the 500 goal, and now the baseline is 110 pods/node.